src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -1,16 +1,15 @@
-@echo off
+@if "%_echo%" neq "on" echo off
 setlocal
 
 set PROJECT_DIR=%~1
 set DOTNET_CMD=%~2
 set TOOLRUNTIME_DIR=%~3
-set PACKAGES_DIR=%4
+set PROJECT_JSON_FILE=%4
```

Comment:
ok, so I'll remove the net45 section of the dependencies and use a new project.json for that.